protocol/messenger_emoji_reactions.go

```diff
@@ -11,7 +11,28 @@ import (
 	"github.com/status-im/status-go/protocol/protobuf"
 )
 
-func (m *Messenger) SendEmojiReaction(ctx context.Context, chatID, messageID string, emojiID protobuf.EmojiReaction_Type) (*MessengerResponse, error) {
+func ConvertEmojiIDToString(emojiID protobuf.EmojiReaction_Type) string {
+	switch emojiID {
+	case protobuf.EmojiReaction_LOVE:
+		return "❤️"
+	case protobuf.EmojiReaction_THUMBS_UP:
+		return "👍"
+	case protobuf.EmojiReaction_THUMBS_DOWN:
+		return "👎"
+	case protobuf.EmojiReaction_LAUGH:
+		return "😂"
+	case protobuf.EmojiReaction_SAD:
+		return "😢"
+	case protobuf.EmojiReaction_ANGRY:
+		return "😠"
+	}
+
+	return ""
+
+}
+
+// TODO remove emojiID once the client supports sending custom emojis
```

Comment:
Once this is done, do we plan to map emoji string to type so that old clients can see at least elementary reactions?